projectile.el

```diff
@@ -3135,9 +3135,11 @@ If PROJECT-ROOT is given, it is opened instead of the project
 root directory of the current buffer file.  If interactively
 called with a prefix argument, the user is prompted for a project
 directory to open."
-  (interactive (list (projectile-completing-read
+  (interactive (and current-prefix-arg
```

Comment:
Won't it better to use `"P"` in interactive and push the optional reading of the the project in the body of the function?

Reply:
That'll also work.  I just thought it would be intuitive and misfire-free to keep interactive code inside of the `(interactive)` construct, as I saw `(projectile-vc)` (non-interactive use without an argument) in projectile.el.  However, I don't feel strongly.  I'll leave it up to you.

Reply:
👍